output/style_output.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Do we want to add these output files or was that unintentional?

Reply:
The style file is generated in the workflow, all output files can be removed before merge